source/support.tex

```diff
@@ -626,6 +626,8 @@
 #define @\defnlibxname{cpp_lib_flat_set}@                          202207L // also in \libheader{flat_set}
 #define @\defnlibxname{cpp_lib_format}@                            202207L // also in \libheader{format}
 #define @\defnlibxname{cpp_lib_format_ranges}@                     202207L // also in \libheader{format}
+#define @\defnlibxname{cpp_lib_formatters}@                        202302L
+  // also in \libheader{version}, \libheader{stacktrace}, \libheader{thread}
```

Comment:
There's no need to say "also in `<version>`", since this is the synopsis of `<version>`.
```suggestion
  // also in \libheader{stacktrace}, \libheader{thread}
```

Reply:
fixed, thanks